proxy/src/main/resources/com/velocitypowered/proxy/l10n/messages_pl_PL.properties

```
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2018 Velocity Contributors
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+velocity.error.already-connected=Jesteś już połączony z tym serwerem!
+velocity.error.already-connected-proxy=Jesteś już połączony z tym proxy!
+velocity.error.already-connecting=Już próbujesz połączyć się z serwerem!
+velocity.error.cant-connect=Nie udało się połączyć z {0}: {1}
+velocity.error.connecting-server-error=Nie udało ci się się połączyć z {0}. Spróbuj ponownie później.
```

When addressing someone, use upper case. Also don't repeat yourself :)
Also reworded slightly to be more in line with English version.
```suggestion
velocity.error.connecting-server-error=Nie udało się Ciebie połączyć z {0}. Spróbuj ponownie później.
```